@@ -1,115 +0,0 @@
<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Stegano</class>
 <widget class="QMainWindow" name="Stegano">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1134</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Stegano</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>120</y>
      <width>261</width>
      <height>441</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="DefaultButton">
       <property name="text">
        <string>Default</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="XORButton">
       <property name="text">
        <string>XOR (Negative)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="RedPlaneButton">
       <property name="text">
        <string>Red Plane</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="GreenPlaneButton">
       <property name="text">
        <string>Green Plane</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="BluePlaneButton">
       <property name="text">
        <string>Blue Plane</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="FullRedButton">
       <property name="text">
        <string>Full Red</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="FullGreenButton">
       <property name="text">
        <string>Full Green</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="FullBlueButton">
       <property name="text">
        <string>Full Blue</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Select file</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>